### PR TITLE
editor: fix editor button validation

### LINKED ISSUE #556: Document editor: save button disabled
**Describe the issue**
The save button is disabled in some case, although no red validation message is displayed. It is therefore not possible to save a record ("document").

**To Reproduce**
1. Log in as a librarian
1. Proceed as follow, either
1.1 Edit https://ils.test.rero.ch/global/documents/1178
1.1 Import ISBN 9782804420611
1.1 Create a new bibliographic record, fill the displayed input fields
1. Ty to save the record

**Expected behavior**
The button is enabled, and the record is saved.

**Additional context**
It may be due to some fields marked as required in the JSON schema but not yet editable (provisionActivity).

### LINKED ISSUE #557: Editor: qualifier vs. note
In the document editor, two subfields of "identifiedBy", note and qualifier, are labelled with the same text "qualifier". In Transifex, the translations are correct.

image

### PR DESCRIPTION
**Pull request by lauren-d**
* Enable/disable FormControl when a fieldset is show/hidden.
* Adapt json schema form and json schema documents.
* Fix save button disabled in document editor.
* Fix identifier subfield labelled with the same text.
* Close #556, Close #557

Co-Authored-by: Johnny Mariéthoz <Johnny.Mariethoz@rero.ch>
Co-Authored-by: Lauren-D <laurent.dubois@itld-solutions.be>

## Why are you opening this PR?

- Implement task 1074
- Fix issue #556 and #557

## How to test?

- ./scripts/bootstrap & ./scripts/setup
- login as librarian
- go to document editor
- import EAN from BNF, edit or create document

## Code review check list

- [ ] Commit message template compliance.
- [ ] Commit message without typos.
- [ ] File names.
- [ ] Functions names.
- [ ] Functions docstrings.
- [ ] Unnecessary commited files?

### CODE REVIEW

File: rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json [hunk 1]
```diff
@@ -114,9 +114,11 @@
     "items": [
       {
         "notitle": true,
+        "description": "",
         "type": "array",
         "items": [
           {
+            "title": "Subject",
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json]: can be done in the jsonschema.

File: rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json [hunk 2]
```diff
@@ -235,9 +237,11 @@
     "items": [
       {
         "notitle": true,
+        "description": "",
         "type": "array",
         "items": [
           {
+            "title": "Abstract",
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json]: can be done in the jsonschema

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json [hunk 3]
```diff
@@ -832,8 +832,8 @@
             "minLength": 1
           },
           "note": {
-            "title": "Qualifier",
```
Comment (jma) [inline rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json]: all these modifications should be also done in the document-minimal jsonschema

**Review by jma — CHANGES_REQUESTED** [review]

File: rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json (hunk 1 above)
Comment (lauren-d) [reply rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json]: done

File: rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json (hunk 3 above)
Comment (lauren-d) [reply rero_ils/modules/documents/jsonschemas/documents/document-v0.0.1_src.json]: done

File: rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json (hunk 2 above)
Comment (lauren-d) [reply rero_ils/modules/documents/jsonschemas/form_documents/document-v0.0.1.json]: done

**Review by jma — APPROVED** [review]

**Pull request merged**